### ISSUE TITLE
implementation of __truediv__

### ISSUE THREAD

**Issue by Freakwill** [gruns/furl#101]
why not implement `__truediv__` to replace method `add`, making things succinct.
```
furl.add(path) == furl / path   # similar to pathlib
```

**Comment by gruns** [comment]
Added in https://github.com/gruns/furl/commit/bb1bcb339e942c1adb3af387ed28eac8cf76cf72.

This will ship in the next version of furl, v1.2.

**Comment by gruns** [comment]
`__truediv__()` support shipped in Furl v1.2.

```python
>>> from furl import furl
>>> f = furl('http://www.google.com/')
>>> f / 'intl' / 'en' / 'about/'
>>> f.url
'http://www.google.com/intl/en/about/'
```

Upgrade with

```
pip install furl --upgrade
```

Thank you for bringing this wonderful suggestion to my attention, @Freakwill.
Don't hesitate to let me know if there's anything else I can do for you.
